.github/workflows/release.yml

```diff
@@ -0,0 +1,35 @@
+name: 🐍 Publish navmazing to PyPI
+
+on:
+  push:
+    branches:
+      - master
+    tags:
+      - '*'
+
+jobs:
+  build-and-publish:
+    name: Build and publish Python 🐍 distributions to PyPI
+    if: startsWith(github.event.ref, 'refs/tags')
+    runs-on: ubuntu-latest
+    steps:
```

Comment:
You can "Add a release step":

```
- name: Create a release
  uses: actions/create-release@v1
  env:
    GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
  with:
    tag_name: ${{ github.ref }}
    release_name: Release ${{ github.ref }}
    draft: false
    prerelease: false
```

Reply:
It will be great. I never tried this action before :).